src/lib.rs

```diff
@@ -141,16 +141,18 @@ impl Tracer {
         self
     }
 
-    // Make the map filename by setting/adding a ".map" extension to the trace filename
-    fn make_map_filename(&self, trace_filename: &CString) -> Result<CString, TraceMeError> {
-        let fn_str = trace_filename.clone().into_string()?;
-        let mut pb = PathBuf::from(fn_str);
+    // Make the map filename by setting/adding a ".map" extension to the trace filename.
+    //
+    // # Arguments
+    //
+    //  * `trace_filename` - The trace filename from which to build the map filename.
+    fn make_map_filename(trace_filename: &str) -> Result<String, TraceMeError> {
```

Comment:
Although I know this function isn't public, I tend to think `///` comments make more sense for function definition comments. BTW, I'm not sure the `#Arguments` style is common for Rust (nor am I sure it is very useful for functions with one argument).

Reply:
I just copied the style of Rust by Example: https://rustbyexample.com/meta/doc.html

Reply:
Yes, but the standard docs don't do it, and I can't think of many Crates that do it unthinkingly. Put another way, if you've got lots of complex parameters ("arguments" is more often used to denote the callee site) I think it can make sense to list them out like this; but it's often more pragmatic to detail the 1 or 2 parameters inline in normal prose.

Reply:
ok